pkg/resourcemanager/appinsights/appinsights.go

```diff
@@ -209,6 +263,10 @@ func (m *Manager) Delete(ctx context.Context, obj runtime.Object, opts ...resour
 
 	if err == nil {
 		if response.Status != "InProgress" {
+			m.DeleteSecret(ctx,
+				i.Spec.ResourceGroup,
+				i.Name,
+				i)
 			return false, nil
```

Comment:
@melonrush13 we now always return nil for the error instead of the err if delete secret fails... this allows for the secret to already be gone